frontend/packages/pipelines-plugin/src/utils/__tests__/hooks.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { PIPELINE_GA_VERSION } from '../../components/pipelines/const';
+import { testHook } from '../../test-data/test-utils';
+import { usePipelineTechPreviewBadge } from '../hooks';
+import * as operatorUtils from '../../components/pipelines/utils/pipeline-operator';
+
+describe('usePipelineTechPreviewBadge:', () => {
+  it('should return the badge if pipeline GA opertaor is installed', () => {
+    jest.spyOn(operatorUtils, 'usePipelineOperatorVersion').mockReturnValue({ version: '1.3.1' });
+    testHook(() => {
+      const badge = usePipelineTechPreviewBadge('test-ns');
+      expect(badge).not.toBeNull();
+      expect(badge).toBeDefined();
```

Comment:
do we need both or just `expect(badge).toBeDefined();` will be enough?

Reply:
removed not.toBeNull() line.